src/components/Contracts/Contracts.tsx

```diff
@@ -0,0 +1,198 @@
+import { ReactComponent as OpenIcon } from "../../img/icons/openIcon.svg"
+import { ReactComponent as CopyIcon } from "../../img/icons/copyIcon.svg"
+import { ReactComponent as UsdtIcon } from "../../img/icons/usdt.svg"
+import { ReactComponent as WbtcIcon } from "../../img/icons/wbtc.svg"
+import { ReactComponent as EthIcon } from "../../img/icons/eth.svg"
+
+const Contracts = () => {
+    return (
+        <div className="container mx-auto w-[350px] rounded-[15px] px-[23px] pt-[29px] pb-[26px] bg-card">
+            <h2 className="font-ubuntu font-medium text-[20px] text-white">Contracts</h2>
+            <div className="mt-[14px] divide-y-[0.4px] divide-solid divide-[#575771]">
+                <div className="h-[29px] pb-[6px] flex items-center justify-between">
+                    <div className="font-ubuntu font-medium text-[12px] text-white">Pool</div>
+                    <div className="flex flex-row">
+                        <span className="font-ubuntu font-medium text-[12px] text-white underline mr-[13px]">
+                            0x96...5590
```

Comment:
This field contains a  crypto address. It needs to take in a value like "0x406794a4e10cd060ba12e731987f23aed6476fe8" and truncate it as shown. And it has to be a link